Source/controls/plrctrls.cpp

```diff
@@ -1495,32 +1546,51 @@ string_view ControlTypeToString(ControlTypes controlType)
 	return "Invalid";
 }
 
+string_view GamepadTypeToString(GamepadLayout gamepadLayout)
+{
+	switch (gamepadLayout) {
+	case GamepadLayout::Generic:
+		return "Unknown";
+	case GamepadLayout::Nintendo:
+		return "Nintendo Switch Pro";
+	case GamepadLayout::Playstation:
+		return "Dual Shock";
+	case GamepadLayout::Virtual:
+		return "Virtual Gamepad";
+	case GamepadLayout::Xbox:
+		return "Xbox";

```

Comment:
Nit: Just for constancy
```suggestion
	case GamepadLayout::Xbox:
		return "Xbox";
	case GamepadLayout::Virtual:
		return "Virtual Gamepad";
```